content/blog/2021-07-16-july-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,247 @@
+﻿---
+title: July '21 Heartbeat
+date: 2021-07-16
+description: |
+  Monthly updates are here! Great new tutorials from the Community, uptick in 
+  jobs requiring DVC, awesome Community discussion on experiments at our June
+  Meetup and a cipher. Can you figure it out?
+descriptionLong: |
+  This month you will find:
+  📈 DVC + Streamlit = ❤️,
+  🇯🇵 DVC in Japanese,
+  📖 A new Udacity Course that includes DVC,
+  🧑🏽‍💻 More and more jobs requiring DVC
+  🧪 June Meetup on Experiments,
+  🚀 New team member, a secret code and more!
+picture: 2021-07-16/july21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/july-heartbeat/825
+tags:
+  - Heartbeat
+  - DVC
+  - CML
+  - Streamlit
+  - Udacity
+---
+
+# Welcome to Summer!
+
+![It's summer!](https://media.giphy.com/media/WuY9yfI89DbNu/giphy.gif)
+
+# From the Community
+
+<span style="color:purple">A</span>s usual we have a ton of goodness from the
```

Comment:
nice touch! :) 
is it intentional? just curious about the idea

Reply:
I got it :)

Reply:
Yes!